tests/helpers/TestCommandLogger.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2018 Ultimaker
+# !/usr/bin/env python
+# -*- coding: utf-8 -*-
+from typing import cast
+from unittest import TestCase
+
+from pymongo.monitoring import CommandStartedEvent, CommandFailedEvent, CommandSucceededEvent
+
+from mongoOperator.helpers.listeners.mongo.CommandLogger import CommandLogger
+
+
+class CommandEventMock:
+    """ Mock implementation of a CommandEvent. """
+    command_name = "foo"
+    request_id = 1
+    connection_id = 1
+    duration_micros = 10000
+
+
+class TestCommandLogger(TestCase):
+    command_logger = CommandLogger()
+
+    def test_started(self):
+        self.command_logger.started(event = cast(CommandStartedEvent, CommandEventMock()))
```

Comment:
Is the cast necessary? A Mock instance doesn't usually need to be casted. Same in the other tests.

Reply:
It's not really needed to run the tests, but PyCharm is showing it as warning, so it's more a prettiness thing.

Reply:
IMO pycharm needs to be configured better, cause this makes the code much less readable; besides it's a lot of nonproductive work